### PR TITLE
Add React and View code snippets to the fieldTemplate article of DDBox

### PR DESCRIPTION
**Pull request by czernous-devexpress**
…DownBox API ref

### CODE REVIEW

**Review by ksercs — APPROVED** [review]

**Comment by ksercs** [comment]
LGTM

File: api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md [hunk 1]
```diff
@@ -3,27 +3,34 @@ id: dxDropDownBox.Options.fieldTemplate
 type: template
 default: null
 ---
+
 ---
```
Comment (AlexJameson) [inline api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md:6]: I guess that somethings wrong here 
image

File: api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md [hunk 2]
```diff
@@ -97,12 +99,72 @@ In the following code, the **fieldTemplate** is used to stylize the text field w
...
+    .custom-item {
+        font-size: 16px;
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDropDownBox...>
+            <template #field="{value}">
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md]: We haven't shown how to specify the `field-template` property.

```suggestion
        <DxDropDownBox ...
            field-template="field">
            <template #field="{ value }">
```

File: api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md [hunk 3]
```diff
@@ -97,12 +99,72 @@ In the following code, the **fieldTemplate** is used to stylize the text field w
...
+            </template>
+        </DxDropDownBox>
+    </template>
+
+    <script>
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import DxDropDownBox from 'devextreme-vue/drop-down-box';
+        import DxTextBox from 'devextreme-vue/text-box';
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md]: - Align imports with the `<script>` tag.
- The declaration of exported components, data, and methods is missing.

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**